src/views/LoggingRoom/LoggingRoomCreate.vue

```diff
@@ -3,9 +3,12 @@
       <div class="logging-room-create__container">
         <BaseCard class="logging-room-create__content">
```

Comment:
BaseCard is a deprecated component, can you please change this to use buefy card or other kind of container instead?